### ISSUE TITLE
Update to quasar v1?

### ISSUE THREAD

**Issue by cmichel21** [eosdac/eosdac-client-legacy#94]
[Quasar v1](https://quasar.dev/) has been released, eosdac-client is still on v0.17.

It seems to have quite some breaking changes. Has there been an effort yet to update to the new version?
I guess it will also break many existing eosdac-client-extensions?

[Update guide](https://quasar.dev/start/upgrade-guide)

**Comment by michaeljyeates** [comment]
We looked at v1 a few months ago and in the end decided that it wasnt worth updating yet.

**Comment by michaeljyeates** [comment]
I think the summary was that the new features werent worth the major breaking changes

**Comment by str** [comment]
What about pushing the upgrade to a next milestone? Sooner or later the update will be needed.

**Comment by piecesnbits** [comment]
yeah I agree with @str it will be needed sooner or later. it's a lot of work though and if we decide to do it then a feature freeze on the client is needed.

**Comment by MrToph** [comment]
I agree that it is necessary to do (unless you plan to rewrite everything in a different framework). A major update is not only about the new features, but having the possibility to easily upgrade in the future in case of security issues / bug fixes / features.
And the longer an update is postponed and new features added on top of the old, the more needs to be done later.

Not to mention the documentation including the search for quasar v1 is so much better than [for 0.17](https://v0-17.quasar-framework.org/components).
